code/modules/projectiles/guns/projectile/CMturret.dm

```diff
@@ -0,0 +1,228 @@
+/mob/living/carbon/human
+	var/obj/structure/machinegun/mounted
+
+	ClickOn(var/atom/A, params)
```

Comment:
You'll need to tell me what to do with it, I'm afraid that prodding it too much will make it all break again.